README.md

```diff
@@ -1,23 +1,55 @@
 # Feedhenry RainCatcher Angularjs
 
-Reference mobile and website implementation for RainCatcher
+Reference mobile and portal implementation for RainCatcher.
 
 ## About RainCatcher Angularjs
 
 RainCatcher Angularjs is reference mobile and website implementation for [Raincatcher Core Framework](https://github.com/feedhenry-raincatcher/raincatcher-core).
 
+Angular.js repository contains set of the modules that implements angular.js based services and directives.
+Modules are used in two different demo applications:
+
+`Demo mobile` application which is used by field engineers to operate and execute items in workflows
+`Demo portal` application which is used by administrators to create workorders and workflows that can be consumed by field engineers.
+
+Both applications require server side demo application, which is located in the core repository.
+
+## Requirements
+
+- node/npm
```

Comment:
That will be in core repository,  but we can duplicate this as well. In my opinion stuff we have is pretty generic and should work with any node version.